common/contract.ts

```diff
@@ -66,6 +66,8 @@ export type Contract<T extends AnyContractType = AnyContractType> = {
   flaggedByUsernames?: string[]
   openCommentBounties?: number
   unlistedById?: string
+
+  dreamImageUrl?: string
```

Comment:
I'd call this `coverImageUrl` or something, because it seems fairly likely that this image will be replaceable by the user for non-dream images (eg unsplash images, or user-uploaded)